index.html

```diff
@@ -279,7 +279,74 @@
       </ul>
     </div>
 
-    <p>If you attend or run a meet-up that isn't here just send us a pull request on <a href="https://github.com/newwwie/newwwie.com" rel="external">github</a> and we can set the record.</p>
+    <div class="card">
+      <h4>
+        <a href="https://www.meetup.com/Newcastle-IoT-Pioneers/">Newcastle IoT Pioneers</a>
+      </h4>
+      <dl>
+        <dt>Links:</dt>
+        <dd>
+          <ul>
+            <li><a href="https://www.meetup.com/Newcastle-IoT-Pioneers/" rel="external">Meetup</a></li>
+          </ul>
+        </dd>
+        <dt>When:</dt>
+        <dd>Monthly, 1st Thurs</dd>
+        <dt>Where:</dt>
+        <dd>Stag and Hunter, Mayfield</dd>
+        <dt>Who:</dt>
+        <dd>In or around Newcastle, Lake Macquarie or the Hunter? Keen to make a splash in the Internet of Things world? Then this Meetup is for you!</dd>
+        <dt>Format:</dt>
+        <dd>Organised presentations.</dd>
+      </dl>
+
+    </div>
+
+    <div class="card">
+      <h4>
+        <a href="https://www.meetup.com/Agile-Newcastle/">Agile Newcastle</a>
+      </h4>
+      <dl>
+        <dt>Links:</dt>
+        <dd>
+          <ul>
+            <li><a href="https://www.meetup.com/Agile-Newcastle/" rel="external">Meetup</a></li>
+          </ul>
+        </dd>
+        <dt>When:</dt>
+        <dd>Varies</dd>
+        <dt>Where:</dt>
+        <dd>Varies</dd>
+        <dt>Who:</dt>
+        <dd>Anyone interested in agile software development practices and methodologies</dd>
+        <dt>Format:</dt>
+        <dd>Organised presentations.</dd>
+      </dl>
+    </div>
+
+    <div class="card">
+      <h4>
+        <a href="https://www.meetup.com/Newcastle-Infracoders/">Newcastle Infracoders and DevOps</a>
+      </h4>
+      <dl>
+        <dt>Links:</dt>
+        <dd>
+          <ul>
+            <li><a href="https://www.meetup.com/Newcastle-Infracoders/" rel="external">Meetup</a></li>
+          </ul>
+        </dd>
+        <dt>When:</dt>
```

Comment:
This group meets erratically.  I think the schedule is defined by the availability of speakers.